scripts/dev_changelog.py

```diff
@@ -250,5 +273,11 @@ def main(token, revision_range, outfile=None):
     parser.add_argument(
         "-o", "--outfile", type=str, help="path and file name of the changelog output"
     )
+    parser.add_argument(
+        "-t", "--tag", type=str, help="the tag of the new release"
+    )
+    parser.add_argument(
+        "-a", "--additional", type=int, action="append", help="include an additional PR in the changelog that has not been recognized automatically. flag can be added more than once."
```

Comment:
I like the idea of this flag, though it might make more sense to use click :P
I know for certain there is a way to take in as many PR nums with just a single instance of -a as say a , separated list rather easily